estuary_updater/handlers/freshmaker.py

```diff
@@ -96,23 +96,27 @@ def create_or_update_build(self, build, event_id):
         """
         Use the Koji Task Result to create or update a ContainerKojiBuild.
 
-        :param dict build: the build being created or updated
+        :param dict build: the build represented in Freshmaker being created or updated
         :param int event_id: the id of the Freshmaker event
         :return: the created/updated ContainerKojiBuild or None if it cannot be created
         :rtype: ContainerKojiBuild or None
         """
+        # build_id in Freshmaker is actually the task_id
         if not build['build_id']:
-            log.debug('Skipping build update for event {0} because no build ID exists.'.format(
+            log.debug('Skipping build update for event {0} because build_id is not set'.format(
                 event_id))
             return None
+
         try:
             koji_task_result = self.koji_session.getTaskResult(build['build_id'])
         except Exception:
             log.error('Failed to get the Koji task result with ID {0}'.format(build['build_id']))
             raise
-        koji_build_id = koji_task_result['koji_builds'][0]
+
+        # The ID is returned as a string so it must be cast to an int
+        koji_build_id = int(koji_task_result['koji_builds'][0])
```

Comment:
optional: I think you can remove the typecasting here and do an `int(koji_build_id)` wherever necessary because it's already being cast to `str()` in certain places.